contracts/IICO.sol

```diff
@@ -76,8 +76,9 @@ contract IICO {
      *  @param _maxBonus The maximum bonus. Will be normalized by BONUS_DIVISOR. For example for a 20% bonus, _maxBonus must be 0.2 * BONUS_DIVISOR.
      *  @param _beneficiary The party which will get the funds of the token sale.
      */
-    function IICO(uint _startTime, uint _fullBonusLength, uint _partialWithdrawalLength, uint _withdrawalLockUpLength, uint _maxBonus, address _beneficiary) public {
+    constructor(uint _startTime, uint _fullBonusLength, uint _partialWithdrawalLength, uint _withdrawalLockUpLength, uint _maxBonus, address _beneficiary) public {
         owner = msg.sender;
+        require(_startTime > now); //sanity check
```

Comment:
Ideally, we want the sale to start after we set up the contract, otherwise we would be wasting part of the full bonus phase.